colormap.py

```diff
@@ -313,6 +313,7 @@
         ('kaust', 'Brown'),
         #( Extras; these are really randomly found user names; needed for screen)
         #( clarity while in color more)
+        ('j', 'Yellow'),       ## yes, let's start editing the colormap file around here ;-)
```

Comment:
why would you insert something so specific in the colormap file for everyone, instead of putting it in your conf file??